src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -0,0 +1,134 @@
+from abc import ABC, abstractmethod
```

Comment:
All of this is worth re-organizing and implementing, but it's a good start for a rough idea of the goal.